examples/robots/lekiwi_client_app.py

```diff
@@ -0,0 +1,144 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from lerobot.common.robots.lekiwi.config_lekiwi import LeKiwiClientConfig, RobotMode
+from lerobot.common.robots.lekiwi.lekiwi_client import LeKiwiClient
+from lerobot.common.teleoperators.keyboard import KeyboardTeleop, KeyboardTeleopConfig
+from lerobot.common.teleoperators.so100 import SO100Leader, SO100LeaderConfig
+
+# TODO(Steven): Check validity of these features
+DUMMY_FEATURES = {
+    "observation.state": {
+        "dtype": "float64",
+        "shape": (9,),
+        "names": {
+            "motors": [
+                "arm_shoulder_pan",
+                "arm_shoulder_lift",
+                "arm_elbow_flex",
+                "arm_wrist_flex",
+                "arm_wrist_roll",
+                "arm_gripper",
+                "base_left_wheel",
+                "base_right_wheel",
+                "base_back_wheel",
+            ]
+        },
+    },
+    "action": {
+        "dtype": "float64",
+        "shape": (9,),
+        "names": {
+            "motors": [
+                "arm_shoulder_pan",
+                "arm_shoulder_lift",
+                "arm_elbow_flex",
+                "arm_wrist_flex",
+                "arm_wrist_roll",
+                "arm_gripper",
+                "base_left_wheel",
+                "base_right_wheel",
+                "base_back_wheel",
+            ]
+        },
+    },
+    "observation.images.front": {
+        "dtype": "image",
+        "shape": (640, 480, 3),
+        "names": [
+            "width",
+            "height",
+            "channels",
+        ],
+    },
+    "observation.images.wrist": {
+        "dtype": "image",
+        "shape": (480, 640, 3),
+        "names": [
+            "width",
+            "height",
+            "channels",
+        ],
+    },
+}
+
+
+def main():
+    logging.info("Configuring Teleop Devices")
+    leader_arm_config = SO100LeaderConfig(port="/dev/tty.usbmodem58760434171")
+    leader_arm = SO100Leader(leader_arm_config)
+
+    keyboard_config = KeyboardTeleopConfig()
+    keyboard = KeyboardTeleop(keyboard_config)
+
+    logging.info("Configuring LeKiwi Client")
+    robot_config = LeKiwiClientConfig(id="lekiwi", robot_mode=RobotMode.TELEOP)
+    robot = LeKiwiClient(robot_config)
+
+    logging.info("Creating LeRobot Dataset")
+
+    # # TODO(Steven): Check this creation
+    # dataset = LeRobotDataset.create(
+    #     repo_id="user/lekiwi2",
+    #     fps=10,
+    #     features=DUMMY_FEATURES,
+    # )
```

Comment:
Should be something like this
```suggestion
    dataset = LeRobotDataset.create(
        repo_id="user/lekiwi",
        fps=10,
        features=robot.features,
    )
```